### ISSUE TITLE
Nextcloud List Folder errors on empty folder

### ISSUE THREAD

**Issue by mvhirsch** [n8n-io/n8n#2057]
**Describe the bug**
List empty folder with nextcloud results in an error, instead of empty array.

**To Reproduce**
1. Prepare an empty nextcloud folder on any nextcloud instance (make sure, it does not contain a readme.md or any other file)
1. Add nextcloud node to workflow
2. List / Folder
3. See error

```
TypeError: jsonResponseData.d:multistatus.d:response is not iterable
    at Object.execute (/data/packages/nodes-base/nodes/NextCloud/NextCloud.node.ts:1102:58)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at /data/packages/core/src/WorkflowExecute.ts:675:26
```

**Expected behavior**
Expected an empty list.

**Environment (please complete the following information):**
 - OS: Fedora 34
 - n8n Version 0.123.1 (Docker)
 - Database system postgres
 - Operation mode queue

**Comment by mvhirsch** [comment]
Reproduceable on `n8nio/n8n:nightly` (Version: 0.132.2).

**Comment by janober** [comment]
Thanks a lot for making us aware of it @mvhirsch ! Got fixed and will be released with the next version later today.

**Comment by janober** [comment]
Fix got released with n8n@0.133.0
